cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,51 @@
-describe('', () => {
-  before(() => {
+describe('Article', () => {
+  let user;
+  let article;
+  beforeEach(() => {
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
 
+    cy.task('newArticle').then((newArticle) => {
+      article = newArticle;
+    });
   });
 
-  it('', () => {
+  it('should provide user to create an article', () => {
```

Comment:
every test should have .should() method to assert the test result

Reply:
i've changed smth in my test, check it pls :)